manifest/1.4.0.xml

```diff
@@ -18,8 +18,8 @@
   
   <project groups="notdefault,sg-accel" name="blance" path="godeps/src/github.com/couchbase/blance" revision="3d39b57188c372649beedd5c13c9003156d5a055" />
   <project name="build" path="cbbuild" revision="388ec72a70fc0925823d726609a209201052567b">
-    <annotation name="RELEASE" value="1.4.0.1" />
-    <annotation name="VERSION" value="1.4.0.1" />
+    <annotation name="RELEASE" value="1.4.0.2" />
```

Comment:
This feels a little confusing to me to have this file named `1.4.0.xml` but it has a release value of `1.4.0.2`.

Would it be possible to have a `manifest/1.4.0.1.xml` with a RELEASE value of `1.4.0.1` and a Would it be possible to have a `manifest/1.4.0.2.xml` with a RELEASE value of `1.4.0.2`

Going to merge for now .. we can discuss async